articles/implementing-shooting-method-in-matlab/index.md

```diff
@@ -0,0 +1,191 @@
+### Shooting method using Matlab
+### Introduction
+
+In numerical analysis, the shooting method reduces a boundary value problem to an initial value problem. It is a popular method for the solution of two-point boundary value problems. If the problem is first defined by *y=f(x); y(x0)=a* and *$y(X_1)$*, it is transformed into the initial problem *y' =z(x)* and *z'(x) = f(x)* with some initial values. Since this method is a two-point boundary value problem, then it has two initial values. In the shooting method, one of the initial values is not known. This value is guessed and compared to the known solution at the boundary conditions until the target thus shooting method.
+
+This article contains the construction of the shooting method code for a linear BVP. We will look at recognizing the boundary value problem and formulate the BVP as an equivalent system of IVP.
+
+### Prerequisites
+
+To follow along with this tutorial, you'll need:
+- [MATLAB](https://www.mathworks.com/products/get-matlab.html?s_tid=gn_getml) installed.
+- Proper understanding of [MATLAB](https://www.section.io/engineering-education/getting-started-with-matlab/) basics.
+
+### Overview
+
+Higher-order ODE's are written as;
+
+$
+\frac{d^n y}{dt^n}= f(t,y,\frac{dy}{dt},\frac{d^2}{dt^2},---,\frac{d^{n-1}y}{dt^{n-1}})
+$
+
+From the above equation, nth order ODEs will require *n* conditions for a unique solution. In the initial value problem, the conditions of the solution function `y` and all its derivatives up to the `N-1`. All at some initial value t=t0. The expression for the equation is;
+
```

Comment:
I'd prefer a situation where you do away with mnemonics and use full definition instead followed by its mnemonic where applicable.